device/device_flasher.go

```diff
@@ -0,0 +1,125 @@
+package device
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/riobard/go-virtualbox"
+	"github.com/xshellinc/iotit/lib/repo"
+	"github.com/xshellinc/iotit/lib/vbox"
+	"github.com/xshellinc/tools/constants"
+	"github.com/xshellinc/tools/lib/help"
+)
+
+// DeviceFlasher is an entity for flashing different devices
+type DeviceFlasher interface {
```

Comment:
__[golint]__
* [golint] type name will be used as device.DeviceFlasher by other packages, and that stutters; consider calling this Flasher

<!-- Comment by [SideCI](https://sideci.com) -->